layouts/shortcodes/members.html

```diff
@@ -2,6 +2,22 @@
 <tbody>
 {{ range $.Site.Data.members.list }}
 <tr><td><h3>{{ .name }}</h3>{{ .intro | markdownify }}</td><td><img src="{{ .image }}" alt="{{ .name }}"/></td></tr>
+<tr><td>
+{{ $pages := (where $.Site.RegularPages ".Params.author" "intersect" (slice .name)) }}
+{{ $pages := $pages | union (where $.Site.RegularPages ".Params.author" .name) }}
+{{ range (where $pages "Section" "!=" "").GroupByDate "2006" }}
+<h4 class="archive-title">{{ .Key }}</h4>
+<ul>
+  {{ range .Pages }}
+  <li>
+    <span class="date">{{ .Date.Format "2006/01/02" }}</span>
+    <a href="{{ .RelPermalink }}">{{ .Title }}</a>
+    <span>{{ .Params.author }}</span>
+  </li>
+  {{ end }}
+</ul>
+{{ end }}
```

Comment:
我觉得对这里的文章列表而言，就不用按年份分组了吧；页面已经很长了，还是用紧凑的方式列举吧：

```suggestion
<ul>
{{ range (where $pages "Section" "!=" "") }}
  <li>
    <span class="date">{{ .Date.Format "2006/01/02" }}</span>
    <a href="{{ .RelPermalink }}">{{ .Title }}</a>
    <span>{{ .Params.author }}</span>
  </li>
{{ end }}
</ul>
```

Reply:
好的，这样看舒服多了。

Reply:
嗯，我觉得不必分组，但可以按日期排序。